support-frontend/assets/pages/[countryGroupId]/router.tsx

```diff
@@ -0,0 +1,25 @@
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import { renderPage } from 'helpers/rendering/render';
+import { geoIds } from 'pages/geoIdConfig';
+import { Checkout } from './checkout';
+
+const router = createBrowserRouter(
+	geoIds
+		.map((geoId) => [
+			{
```

Comment:
We'll add the `/${geoId}` soon as part of the migration from `/{geoId}/contribute` work as this URL is currently redirected via Fastly and the Play app.